backends/cuda/ceed-cuda.c

```diff
@@ -23,6 +23,7 @@
 int compile(Ceed ceed, const char *source, CUmodule *module,
             const CeedInt numopts, ...) {
   int ierr;
+  cudaFree(0);//Make sure a Context exists for nvrtc
```

Comment:
Is the usual/recommended fix?

Reply:
Surprisingly it seems that it is the usual fix: https://stackoverflow.com/questions/10415204/how-to-create-a-cuda-context

Reply:
Huh. CUDA can be weird. Thanks for the info!